docs/programming/tilebuffer.md

```diff
@@ -0,0 +1,285 @@
+# Multi-directional Scrolling (TileBuffer)
+
+## Basic description
+
+This tutoriel only focus on multi-directional scolling as used for RPG or games like Alien Breed, Chaos Engine or large platform game like Turrican or Superfrog.
+
+In order to do this multi-directional scolling, in ACE you need to use  the [managers/viewport/tilebuffer.h](../../include/ace/managers/viewport/tilebuffer.h). It manages for tiled background/map as a main entry.
+
+## Prerequisites
+
+The tile manager takes 2 pre-requisites :
+
+1. a set of tiles
+2. a map using this tiles.
+
+Last pre-requisites, you need the *palette* linked to your tiles graphic. [See Palette Tutorial](./palette.md)
+
+### Prepare the set of tiles
+
+Let's use this ressource from open game art : https://opengameart.org/content/overworld-grass-biome
+
+Get `TilesetGrass/overworld_tileset_grass.png` and convert it in 32 colors (use [grafx2](http://grafx2.chez.com/) for example).
+
+Now using [aseprite](https://www.aseprite.org/]).
+
+Do `File > Import > Import Sprite Sheet`  it in asesprite in 16x16 sprites.
+
+Apply plugin [Aseprite Script: Amiga OCS/ECS Color Palette Mixe](https://prismaticrealms.itch.io/aseprite-script-amiga-ocsecs-color-palette-mixer) to align png and palette to amiga capacity/rgb colors.
+
+Then do `File > Export >  Export Sprite Sheet` and then `File > Saves As` and save as `overworld.png`.
```

Comment:
Not sure about my suggestion because I don't have aseprite and can't really tell if "16x16" is an option in that dialog.

again, not sure if we should go with 1-2-3 steps or 1-1-1.

```suggestion
1. Select `File > Import > Import Sprite Sheet`. Select 16x16 sprites option.
2. Use plugin [Aseprite Script: Amiga OCS/ECS Color Palette Mixe](https://prismaticrealms.itch.io/aseprite-script-amiga-ocsecs-color-palette-mixer) to align png and palette to amiga capacity/rgb colors.
3. Select `File > Export >  Export Sprite Sheet` and then `File > Saves As` and save as `overworld.png`.
```

Reply:
Oh missed this one. On it.